spec/classes/init_spec.rb

```diff
@@ -702,4 +761,63 @@
       end
     end
   end
+
+  describe 'with enable_driftfile set to' do
```

Comment:
Changed the way it works. If $driftfile is empty the complete section will not be used anymore.
Because it's not mandatory but desireable to turn of driftfile for Solaris I have not touched the default values anymore. But spec tests are prepared for such a default in future times.